fog/ledger/server/tests/router_integration.rs

```diff
@@ -40,6 +40,21 @@ const POLL_INTERVAL: Duration = Duration::from_millis(10);
 const TEST_URL: &str = "http://www.my_url1.com";
 const CHAIN_ID: &str = "local";
 
+fn assert_key_image_unspent(key: &KeyImage, result: &KeyImageResult) {
+    assert_eq!(result.key_image, *key);
+    // None in the status means not spent.
+    assert_eq!(result.status(), Ok(None));
```

Comment:
This used to have 
```rust
    assert_eq!(
        result.timestamp_result_code,
        TimestampResultCode::TimestampFound as u32
    );
```
like below, it was copied from the original test logic, however after looking over more the timestamp code applies to the spent_at value in the proto file. The spent at should only be populated if the key image is found, this is what `status()` does behind the scenes. As such verifying the timestamp result doesn't apply